### ISSUE TITLE
Detect PIR sensor as 'motion' (again)

### ISSUE THREAD

**Issue by fguerzoni** [davesmeghead/visonic#26]
I've a couple of PIR sensors that are identified as 'magnet' instead of 'motion'.
I found in the code the following warning:
```
# User cybfox77 found that PIR sensors were returning the sensor type 'sensorID_c' as 0xe5 and 0xd5, these would be decoded as Magnet sensors
# This is a very specific workaround for that particular panel type and model number and we'll wait and see if other users have issues
#          These issues could be either way, users with or without that panel/model getting wrong results
#          [handle_msgtype3C] PanelType=4 : PowerMax Pro Part , Model=62   Powermaster False
```
and I think it could be useful to let you know my use case.
So those PIRs, that look similar to others, are associated to sensorID_c = 213 that results to 5.
The ones which are detected as 'motion' have either sensorID_c = 227 or sensorID_c = 195 that results to 3.

My panel is:
[handle_msgtype3C] PanelType=1 : PowerMax+ , Model=32   Powermaster False

The following pastebin reports the full log with some custom rows I inserted for debugging
https://pastebin.com/HqFvdjUL

**Comment by davesmeghead** [comment]
A couple of quick questions please:
1. Are all your motion sensors the same model number on the label?
2. Are all your motion sensors registered in the panel as the same?  I'm not sure how to check this on a powermax+.

There is a quick fix but I'd rather get the answers to the above and do it properly.  If they are all the same PIR model number then they must be registered in the panel in a different way for them to be reported as magnet sensors.

And now on to the quick fix, will changing this line
```
if self.PanelType == 4 and self.ModelType == 62 and sensorID_c in powermax_pro_sensortypes:
```
to this
```
if ((self.PanelType == 4 and self.ModelType == 62) or (self.PanelType == 1 and self.ModelType == 32)) and sensorID_c in powermax_pro_sensortypes:
```
Fix it for you?

I realise this is a quick fix but without knowing all the panel type and sensor type combinations then it's pretty difficult.

**Comment by fguerzoni** [comment]
The fix runs fine, now all PIR detectors are seen as 'motion'.
Thank you

PIRs have both same model number
- NEXT MCW/K9 MCW
- 8-3591-A20 v.01

The only difference I notice is the production date:
- 01/11 on the sensor seen as 'magnet'
- 05/10 on the sensor seen as 'motion'

Please see the picture from the link where I put both one sensor seen as 'magnet' and one sensor seen as 'motion'
https://imagebin.ca/v/5n4hD0FF9MBc

All the sensors are registered the same on the Powermax+ panel.
Basically you have to register a sensor to a zone,
and then you have to specify the zone type (13-volumetric) and name(from a predefined menu)

**Comment by davesmeghead** [comment]
The release of version 0.6.4.0 should include this fix so I'm closing the issue,
